### ISSUE TITLE
Help needed with deployment on heroku

### ISSUE THREAD

**Issue by AustrianBurners** [Edgeryders-Participio/dreams#114]
I had to change the github repository for some reason. I also created a new main branch as being told by heroku and pushed it. I linked the github account to heroku but deployment fails with errors. I do not understand the solution in the log "Run `bundle install` elsewhere ". WHERE should I run that? I have no idea how this GEMthingies work!

`-----> Building on the Heroku-18 stack
-----> Using buildpack: heroku/ruby
-----> Ruby app detected
-----> Installing bundler 2.2.16
-----> Removing BUNDLED WITH version in the Gemfile.lock
-----> Compiling Ruby/Rails
###### WARNING:
       Your app was upgraded to bundler 2.2.16.
       Previously you had a successful deploy with bundler 2.0.2.
       
       If you see problems related to the bundler version please refer to:
       https://devcenter.heroku.com/articles/bundler-version#known-upgrade-issues
       
-----> Using Ruby version: ruby-2.5.1
-----> Installing dependencies using bundler 2.2.16
       Running: BUNDLE_WITHOUT='development:test' BUNDLE_PATH=vendor/bundle BUNDLE_BIN=vendor/bundle/bin BUNDLE_DEPLOYMENT=1 bundle install -j4
       You are trying to install in deployment mode after changing
       your Gemfile. Run `bundle install` elsewhere and add the
       updated Gemfile.lock to version control.
       
       If this is a development machine, remove the /tmp/build_0d31ef42/Gemfile freeze 
       by running ``.
       
       The list of sources changed
       
       You have added to the Gemfile:
       * source: https://github.com/activeadmin/activeadmin.git
       
       You have deleted from the Gemfile:
       * source: git://github.com/activeadmin/activeadmin.git
       Bundler Output: You are trying to install in deployment mode after changing
       your Gemfile. Run `bundle install` elsewhere and add the
       updated Gemfile.lock to version control.
       
       If this is a development machine, remove the /tmp/build_0d31ef42/Gemfile freeze 
       by running ``.
       
       The list of sources changed
       
       You have added to the Gemfile:
       * source: https://github.com/activeadmin/activeadmin.git
       
       You have deleted from the Gemfile:
       * source: git://github.com/activeadmin/activeadmin.git
 !
 !     Failed to install gems via Bundler.
 !
 !     Push rejected, failed to compile Ruby app.
 !     Push failed`

**Comment by aerugo** [comment]
Hi! This is the old version of Dreams, and it is no longer being maintained. We have a new version now that is hosted and a lot better. You can try it out here: https://dreams.wtf/

Here is an example of it in action:
https://kiezburn.dreams.wtf/kb21

**Comment by aerugo** [comment]
We should probably update the README with this info now that we’ve stopped maintaining.

**Comment by aerugo** [comment]
Our new version is also open-source, but it’s a little more complicated to set up.

https://github.com/Edgeryders-Participio/multi-dreams

**Comment by AustrianBurners** [comment]
yes, but I have everything running! I just cannot deploy the update!

**Comment by AustrianBurners** [comment]
I did this now:
heroku run "bundle install"  -a schoenburn-dreams
and it succeded:
Bundle complete! 53 Gemfile dependencies, 155 gems now installed.
Gems in the groups development and test were not installed.
Bundled gems are installed into `./vendor/bundle`

but still, deployment is not possible. How do I solve this gemfile thing?

```
You have added to the Gemfile:
source: https://github.com/activeadmin/activeadmin.git   
You have deleted from the Gemfile:
source: git://github.com/activeadmin/activeadmin.git
```

**Comment by AustrianBurners** [comment]
git://github.com/activeadmin/activeadmin.git is obviously a totally different project. ???

**Comment by aerugo** [comment]
No idea, unfortunately. We moved away from Ruby a couple of years ago, and the new version is on a more modern JavaScript stack.

ActiveAdmin is the admin backend component that was used with the old Dreams platform.

I think that it probably requires a bit of work on the code to get the old Dreams platform to work with the current versions of Ruby and Rails that are required for Heroku. My recommendation would be to rather use the new Dreams platform - either on the dreams.wtf deployment or self hosted.

**Comment by AustrianBurners** [comment]
ok. so I will dump it.
